2024-08-09 06:19:46,666 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.34.59.6:44805'
2024-08-09 06:19:46,670 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.34.59.6:34753'
2024-08-09 06:19:47,736 - distributed.worker - INFO -       Start worker at:     tcp://10.34.59.6:43232
2024-08-09 06:19:47,736 - distributed.worker - INFO -          Listening to:     tcp://10.34.59.6:43232
2024-08-09 06:19:47,736 - distributed.worker - INFO -           Worker name:           SLURMCluster-1-1
2024-08-09 06:19:47,737 - distributed.worker - INFO -          dashboard at:           10.34.59.6:45049
2024-08-09 06:19:47,737 - distributed.worker - INFO - Waiting to connect to:     tcp://10.34.59.1:44816
2024-08-09 06:19:47,737 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:19:47,737 - distributed.worker - INFO -               Threads:                          1
2024-08-09 06:19:47,737 - distributed.worker - INFO -                Memory:                   1.86 GiB
2024-08-09 06:19:47,737 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-a3s74l7u
2024-08-09 06:19:47,737 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:19:47,740 - distributed.worker - INFO -       Start worker at:     tcp://10.34.59.6:38227
2024-08-09 06:19:47,740 - distributed.worker - INFO -          Listening to:     tcp://10.34.59.6:38227
2024-08-09 06:19:47,740 - distributed.worker - INFO -           Worker name:           SLURMCluster-1-0
2024-08-09 06:19:47,740 - distributed.worker - INFO -          dashboard at:           10.34.59.6:35160
2024-08-09 06:19:47,740 - distributed.worker - INFO - Waiting to connect to:     tcp://10.34.59.1:44816
2024-08-09 06:19:47,740 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:19:47,740 - distributed.worker - INFO -               Threads:                          1
2024-08-09 06:19:47,740 - distributed.worker - INFO -                Memory:                   1.86 GiB
2024-08-09 06:19:47,740 - distributed.worker - INFO -       Local Directory: /tmp/dask-scratch-space/worker-fjekhi0r
2024-08-09 06:19:47,740 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:19:48,083 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-08-09 06:19:48,084 - distributed.worker - INFO -         Registered to:     tcp://10.34.59.1:44816
2024-08-09 06:19:48,084 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:19:48,084 - distributed.core - INFO - Starting established connection to tcp://10.34.59.1:44816
2024-08-09 06:19:48,085 - distributed.worker - INFO - Starting Worker plugin shuffle
2024-08-09 06:19:48,085 - distributed.worker - INFO -         Registered to:     tcp://10.34.59.1:44816
2024-08-09 06:19:48,085 - distributed.worker - INFO - -------------------------------------------------
2024-08-09 06:19:48,086 - distributed.core - INFO - Starting established connection to tcp://10.34.59.1:44816
2024-08-09 06:20:06,704 - distributed.worker - INFO - Stopping worker at tcp://10.34.59.6:43232. Reason: scheduler-close
2024-08-09 06:20:06,704 - distributed.worker - INFO - Stopping worker at tcp://10.34.59.6:38227. Reason: scheduler-close
2024-08-09 06:20:06,705 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Worker->Scheduler local=tcp://10.34.59.6:40094 remote=tcp://10.34.59.1:44816>
Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 297, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/tornado/gen.py", line 767, in run
    value = future.result()
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 307, in write
    convert_stream_closed_error(self, e)
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Worker->Scheduler local=tcp://10.34.59.6:40094 remote=tcp://10.34.59.1:44816>: Stream is closed
2024-08-09 06:20:06,705 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Worker->Scheduler local=tcp://10.34.59.6:40096 remote=tcp://10.34.59.1:44816>
Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 297, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/tornado/gen.py", line 767, in run
    value = future.result()
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 307, in write
    convert_stream_closed_error(self, e)
  File "/users/stud/k/kruret00/.local/lib/python3.10/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Worker->Scheduler local=tcp://10.34.59.6:40096 remote=tcp://10.34.59.1:44816>: Stream is closed
2024-08-09 06:20:06,708 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://10.34.59.6:34753'. Reason: scheduler-close
2024-08-09 06:20:06,708 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://10.34.59.6:44805'. Reason: scheduler-close
slurmstepd: error: *** JOB 2330457 ON cl-node006 CANCELLED AT 2024-08-09T06:20:06 ***
2024-08-09 06:20:06,710 - distributed.core - INFO - Received 'close-stream' from tcp://10.34.59.1:44816; closing.
2024-08-09 06:20:06,710 - distributed.nanny - INFO - Worker closed
2024-08-09 06:20:06,711 - distributed.core - INFO - Received 'close-stream' from tcp://10.34.59.1:44816; closing.
2024-08-09 06:20:06,711 - distributed.nanny - INFO - Worker closed
